src/pages/Experience/Experience.js

```diff
@@ -1,9 +1,146 @@
 import React from 'react';
+import OpenSourceCard from './open-source-card';
 
 const Experience = () => {
+    const openSourcePrograms = [
+        {
+          title: "GirlScript Summer Of Code 2020",
+          cardImage: "src/assets/images/open-sorce/1.jpg",
+          description:
+            "Responsible for handling the projects GirlScript App and GirlScript Website Boilerplate.",
+        },
+        {
+          title: "StudentCode-in 2020",
+          cardImage: "src/assets/images/open-source/2.jpg",
```

Comment:
Have a look at the above reference for the structure of pages and the corresponding sections.

Restructure this into the open-source section. 

Try to use JSX